src/viewport/viewport.util.spec.ts

```diff
@@ -1,29 +1,15 @@
-import { isViewportConditionMatch } from "./viewport.util";
-import { ViewportSizeType, ViewportSizeTypeInfo, ComparisonOperation, } from "./viewport.model";
-
-export const sizeRefs = {
-	xsmall: Object.freeze({
-		name: "xsmall",
-		type: ViewportSizeType.xsmall,
-		widthThreshold: 450,
-	} as ViewportSizeTypeInfo),
-	small: Object.freeze({
-		name: "small",
-		type: ViewportSizeType.small,
-		widthThreshold: 767,
-	} as ViewportSizeTypeInfo),
-	medium: Object.freeze({
-		name: "medium",
-		type: ViewportSizeType.medium,
-		widthThreshold: 992,
-	} as ViewportSizeTypeInfo),
-	large: Object.freeze({
-		name: "large",
-		type: ViewportSizeType.large,
-		widthThreshold: 1200,
-	} as ViewportSizeTypeInfo),
-};
-
+import { isViewportConditionMatch, generateViewportDictionary } from "./viewport.util";
+import { ViewportSizeType, ComparisonOperation, } from "./viewport.model";
+
+export const sizeRefs = generateViewportDictionary({
```

Comment:
Updated test to use the new util method used to generate the static information around breakpoints